### PR TITLE
Add yaml support.

### PR DESCRIPTION
**Pull request by EvgeneOskin**

### CODE REVIEW

File: swagger_spec_validator/common.py [hunk 1]
```diff
@@ -24,8 +25,16 @@ def wrapper(*args, **kwargs):
 
 
 def load_json(url):
+    _load(url, json.loads)
+
+
+def load_yaml(url):
+    _load(url, yaml.loads)
```
Comment (sjaensch) [inline swagger_spec_validator/common.py]: This should be yaml.safe_load if I'm not mistaken.

**Comment by sjaensch** [comment]
Hi @EvgeneOskin, thank you for your contribution! Currently the build is red, will you be fixing it? That would be awesome.
Also, there might be issues with specs in the yaml format that don't quote the response status code (#49). You might want to add a workaround similar to what was done for bravado.

**Comment by EvgeneOskin** [comment]
Hi @sjaensch, thank you for review and notes, I've fixed the build. Add simple test to load yml and json data. Also, I've fixed paths in `validator12.run_test`. At the moment, the build fails only in doc environment.

File: swagger_spec_validator/util.py [hunk 2]
```diff
@@ -48,6 +49,12 @@ def validate_spec_url(spec_url):
                        as `file://` this must be an absolute url for
                        cross-refs to work correctly.
     """
-    spec_json = load_json(spec_url)
+    _, extension = os.path.splitext(spec_url)
+    is_yaml = any([i == extension for i in ('.yml', '.yaml')])
```
Comment (dimbleby) [inline swagger_spec_validator/util.py]: why not:
```python
is_yaml = extension in ['.yml', '.yaml']
```
?

Comment (EvgeneOskin) [inline swagger_spec_validator/util.py]: You're right. I've improved it.

**Comment by larsbutler** [comment]
@EvgeneOskin @sjaensch This branch appears to just need a rebase on the latest master, due to https://github.com/Yelp/swagger_spec_validator/commit/a4669b8dd05391fdb528d90af974b7ee7a003f6f. After that, it should work.

**Comment by EvgeneOskin** [comment]
Thank @larsbutler

**Comment by larsbutler** [comment]
@EvgeneOskin @sjaensch Hmmm, something seems to be stuck with coveralls. The check is still pending. It'd be great to get this patch in; I would use it right away. :)

**Pull request closed without merging**